lib/run-status.js

```diff
@@ -126,6 +128,17 @@ RunStatus.prototype.handleTest = function (test) {
 	this.emit('test', test, this);
 };
 
+RunStatus.prototype.formatAssertionError = function (error) {
+	if (error.actual && error.expected) {
+		return '\n    ' + chalk.red(error.name + ': ' + error.message) + '\n' +
+			chalk.green('    + expected') + chalk.red(' - actual') + '\n\n' +
+			chalk.red('    -' + error.actual) + '\n' +
+			chalk.green('    +' + error.expected);
+	}
+
+	return chalk.red('\n    ' + error.name + ': ' + error.message);
```

Comment:
Done. :+1:
I assumed not necessary create a new test/run-status.js, since the added tests already check RunStatus.formatAssertionError() output. Right? :smiley: